README.md

```diff
@@ -1 +1,62 @@
-# bookstore
\ No newline at end of file
+# Bookstore
+
+This is the third weekly project in the react Module. The project involves a simple Bookstore that adds and removes books from an  external API
+
+![screenshot(./images/scn.png)]
```

Comment:
- [ ] I recommend adding a project screenshot because it gives one of the first imagination to users